presto-hive/src/main/java/io/prestosql/plugin/hive/HiveColumnHandle.java

```diff
@@ -58,8 +59,8 @@
 
     public static final int FILE_MODIFIED_TIME_COLUMN_INDEX = -14;
     public static final String FILE_MODIFIED_TIME_COLUMN_NAME = "$file_modified_time";
-    public static final HiveType FILE_MODIFIED_TIME_TYPE = HIVE_LONG;
-    public static final Type FILE_MODIFIED_TIME_TYPE_SIGNATURE = BIGINT;
+    public static final HiveType FILE_MODIFIED_TIME_TYPE = HiveType.HIVE_TIMESTAMP;
```

Comment:
Add 
```
// TODO introduce HiveType.HIVE_TIMESTAMP_WITH_TIME_ZONE
```

-- we cannot add this type here before we upgrade to Hive 3.1, but we should eventually clean this up